zipline/algorithm.py

```diff
@@ -531,6 +531,19 @@ def _create_clock(self):
             # as the last minute of the session.
             market_opens = market_closes
 
+        # The calendar's execution times are the minutes over which we actually
+        # want to run the clock. Typically the execution times simply adhere to
+        # the market open and close times. In the case of the futures calendar,
+        # for example, we only want to simulate over a subset of the full 24
+        # hour calendar, so the execution times are used to artificially create
```

Comment:
I guess the word artificial doesn't fit here anymore, it made more sense when I was calculating the timedeltas here because we were manually interfering with the simulation times. But since the behavior was moved, it's no longer like an interference, it's just part of the calendar interface